packages/web-pkg/src/composables/actions/files/useFileActionsOpenShortcut.ts

```diff
@@ -67,12 +66,11 @@ export const useFileActionsOpenShortcut = () => {
       label: () => {
         return $gettext('Open shortcut')
       },
-      isEnabled: ({ resources }) => {
+      isVisible: ({ resources }) => {
         if (
           unref(isFilesAppActive) &&
           !unref(isSearchActive) &&
           !isLocationSpacesActive(router, 'files-spaces-generic') &&
-          !isLocationPublicActive(router, 'files-public-link') &&
```

Comment:
What's a reason for removing this check?

Reply:
I have no clue how this happened